group/assignment5/main.cpp

```diff
@@ -0,0 +1,96 @@
+#include "AlphabetFromDictionary.hpp"
+#include "codeu_test_lib.h"
+
+int main() {
+    vector<string> dictionary;
+    vector<char> alphabet;
+    bool result;
+    unordered_map<char, int> idxs;
+    vector<bool> expectedChars;
+    vector<bool> actualChars;
+
+    // test 1 - assignment example
+    dictionary = {"ART", "RAT", "CAT", "CAR"};
+    AlphabetFromDictionary afd(dictionary);
+    idxs.clear();
```

Comment:
The same comment as for _afd_ name. Maybe _idxs_ means something? It's hard to read code with this short names, because each time you need to search for its definition.

Reply:
I agree about _afd_ and _idxs_ names, they are a bit confusing. When you read _main.cpp_ for the first time, it's not clear what kind of indexes excatly are there.